pkg/types/vsphere/platform.go

```diff
@@ -66,6 +67,8 @@ type Platform struct {
 	// Network specifies the name of the network to be used by the cluster.
 	Network string `json:"network,omitempty"`
 
-	// Disk Type Thin specifies if thin disks should be use instead of thick
+	// DiskType is the name of the disk provisioning type
+	// for vsphere, valid values are thin, thick, or eagerZeroedThick.
+	// The string is optional.
```

Comment:
Please describe the behavior when the field is omitted.
```suggestion
	// When omitted, ...
	// +optional
```